packages/react-components/react-combobox/library/src/components/Combobox/useComboboxStyles.styles.ts

```diff
@@ -33,6 +33,7 @@ const useStyles = makeStyles({
     justifyContent: 'space-between',
     minWidth: '250px',
     position: 'relative',
+    verticalAlign: 'middle',
```

Comment:
Any ideas as to why this one line change increases the bundle size for this component so much?

Reply:
I don't think it's related to the current PR, the bundle size metrics appear to be fine after rebasing with the latest master.

<img width="789" alt="Screenshot 2024-07-02 at 12 52 10" src="https://github.com/microsoft/fluentui/assets/1492102/5f38d34c-e3d1-460e-800b-0aa159f9b26c">